tasks/task02/TriangleArea.py

```diff
@@ -0,0 +1,40 @@
+# Function to check if number is narcissistic
+def printmenu():
+    print("")
+    print("Menu:")
+    print("1. Calculate triangle area by base and height")
+    print("2. Calculate triangle area by 2 sides and angle between them")
+    print("3. Exit")
+
+
+def calch():
+    s = input("Enter base and height: ")
+    parts = s.split(" ")
+    base = float(parts[0])
+    height = float(parts[1])
+    areaFloat = 0.5*base*height
+    print(f"Area is: {round(areaFloat)}")
+
+
+def calca():
+    import math
+    s = input("Enter 2 sides and angle(degrees) between them: ")
+    parts = s.split(" ")
+    sideA = float(parts[0])
+    sideB = float(parts[1])
+    angle = int(parts[2])
```

Comment:
тут очень удачно подходит тот способ что показывал Семен: распаковка + map
```suggestion
    sideA, sideB, angle = map(int, parts)
```